views/layouts/HeroLayout.css

```diff
@@ -0,0 +1,12 @@
+.HeroLayout_container {
+  display: flex;
+  flex-direction: column;
+  height: 897px;
```

Comment:
why are we specifying a pixel height? Should the layout take up at least the screen height?

Reply:
[9d7c4dc](https://github.com/AMP-SCZ/dpdash/pull/468/commits/9d7c4dc05024fa07956aaa578a84f844f590db07)